src/middleware/auth/base-auth.ts

```diff
@@ -59,24 +59,25 @@ export abstract class AbstractAuthHandler implements IAuthResourceHandler
         if (rule === null) {
             return this.returnError(500, `Internal error. Issue with finding the rule for the path ${request.path}`)
         } else {
-            // Tries to get token from the request and other preps
-            const _setup = this.setupAuthToken(request)
-
-            // If token is not provided - try to proceed without it
-            // If token is provided - verify it and get the scopes
-            if (_setup.status !== 200) {
-                return _setup 
+            // If the user is not authenticated - return error
+            if (!request.user.isAuthenticated) {
+                return this.returnError(400, "Unauthorized error: Seems like you are not authenticated. Please follow the authentication process using 'LogIn' button")
             }
-            const resourceAPI = AbstractAuthHandler.buildResourceAPIUrl(request)
-            if (!resourceAPI) {
-                return this.returnError(500, `Internal error. Issue with building resource API for the path ${request.path}`)
+            // Tries to get customerId from the logTo user structure
+            if (request.user && request.user.claims) {
+                this.customerId = request.user.claims.sub
+            } else {
+                return this.returnError(500, "Unauthorized error: Seems like authentication process was corrupted and there are problems with getting customerId")
```

Comment:
if we are reusing, would be good to store them as error message constants

Reply:
It's a unique error message :) Or do you mean that we need to define all the error messages as constants?

Reply:
I like the thinking here @DaevMithran but storing the error message text somewhere else could make the error return less obvious when reading code (since I'd then have to look up the constants file). A better idea is to use a convention/format when writing error messages, rather than picking a random text (but keep it within the context of where the error is sent from).

So I don't think you need to move this to a constants file, but you guys do need to get consistent with what HTTP status codes you use. :) Since this is true in a couple of other places, I'll leave it as an overall comment.